rekcurd_dashboard/apis/api_evaluation.py

```diff
@@ -109,6 +152,36 @@ def delete(self, project_id: int, application_id: str, evaluation_id: int):
         return {"status": True, "message": "Success."}
 
 
+@evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluations/<int:evaluation_id>/download')
+class ApiEvaluationIdDownload(Resource):
+    @evaluation_api_namespace.response(200, description='return evaluation text file.')
+    @evaluation_api_namespace.produces(['text/plain'])
+    def get(self, project_id: int, application_id: str, evaluation_id: int):
```

Comment:
Could you add `@evaluation_api_namespace.marshal_with` here?

Reply:
This API returns text file, not JSON, so probably there is no way to add marshal_with

(At least, I could not find it..)

Reply:
I see.
(I also could not find the way...)